packages/networks/src/endpoints.ts

```diff
@@ -1,38 +1,49 @@
 import { NetworkEndpoints } from './types'
 
 export const endpointsMainnetK8s: NetworkEndpoints = {
-  indexer: 'https://k8s.mainnet.exchange.grpc-web.injective.network',
-  grpc: 'https://k8s.mainnet.chain.grpc-web.injective.network',
-  rpc: 'https://k8s.mainnet.tm.injective.network',
-  rest: 'https://k8s.mainnet.lcd.injective.network',
+  indexer:
```

Comment:
Can you keep old endpoints (without `global`) just add chronos and explorer, so it would be like  
```
export const endpointsMainnetK8s: NetworkEndpoints = {
  indexer: 'https://k8s.global.mainnet.global.exchange.grpc-web.injective.network',
  grpc: 'https://k8s.mainnet.chain.grpc-web.injective.network',
  rpc: 'https://k8s.mainnet.tm.injective.network',
  rest: 'https://k8s.mainnet.lcd.injective.network',
  chronos: 'https://k8s.mainnet.chronos.grpc-web.injective.network',
  explorer: 'https://k8s.mainnet.explorer.grpc-web.injective.network',
}
```
Then add one more new network type called `endpointsMainnetLB` with those values 

```
export const endpointsMainnetLB: NetworkEndpoints = {
  indexer:'https://k8s.global.mainnet.global.exchange.grpc-web.injective.network',
  grpc: 'https://k8s.global.mainnet.chain.grpc-web.injective.network',
  rpc: 'https://k8s.global.mainnet.tm.injective.network',
  rest: 'https://k8s.global.mainnet.lcd.injective.network',
  chronos: 'https://k8s.global.mainnet.chronos.grpc-web.injective.network',
  explorer: 'https://k8s.global.mainnet.explorer.grpc-web.injective.network',
}
```

This way we can support both K8s and global LB for mainnet (we don't have explorer here yet)